drivers/hmis/app/models/hmis/ce/configuration.rb

```diff
@@ -0,0 +1,41 @@
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# accessor API for CE configuration
+module Hmis::Ce
+  class Configuration
+    # feature flag for CE
+    def enabled?
+      !!value_for(:enabled)
+    end
+
+    # If a candidate pool is no longer associated with any active opportunities, how long should it be retained?
+    def days_to_retain_orphan_candidate_pools
+      value_for(:days_to_retain_orphan_candidate_pools)&.to_i
+    end
+
+    protected
+
+    # read all configuration values from the db
+    PROPERTIES = [
+      :enabled,
+      :days_to_retain_orphan_candidate_pools,
+    ].freeze
+    def values
+      @values ||= AppConfigProperty.
```

Comment:
This is great.  I thought we had a solution but couldn't remember what it was.  
Only note I have is that CE will probably need to be turned on per-data source as we might have multiple HMISs backed by one warehouse.  Not sure if that should get integrated in the key or value, or somehow else.

Reply:
Oh okay, if it's configured per data source I'd be inclined to put the config flag on the data_source table instead. The AppConfigProperty model is helpful for global config but probably better to just the more specific models if possible

Reply:
In thinking about this some more, I'm not sure we need it per data source.  I think the ability to enable/disable CE is probably more during the build-out than once it's ready.  Once it's ready, we can control access to the features via permissions or project setup.  This is probably fine as is.